README.md

```diff
@@ -1,3 +1,159 @@
-# React Native Elements Playground
+<p align="center">
+  <a href="https://reactnativeelements.com/">
+    <img alt="react-native-elements" src="https://user-images.githubusercontent.com/5962998/65694309-a825f000-e043-11e9-8382-db0dba0851e3.png" width="300">
+  </a>
+</p>
+
+<p align="center">
+<b><a href="https://reactnative.dev">React Native Elements</a> Playground 🚀 </b> <br/>
+   React Native elements Playground is An interactive component playground. Which allows you to interact with different React Native Element Components.
+</p>
+<hr/>
+<p align="center">
+  <a href="https://www.npmjs.com/package/react-native-elements"><img src="https://img.shields.io/npm/v/react-native-elements.svg"></a>
+  <a href="https://github.com/react-native-elements/react-native-elements"><img src="https://img.shields.io/github/stars/react-native-elements/playground"></a>
+  <a href="https://react-native-elements-slack.herokuapp.com"><img src="https://react-native-elements-slack.herokuapp.com/badge.svg"></a>
+</p>
+
+<p align="center">
+  <a href="#backers"><img src="https://opencollective.com/react-native-elements/backers/badge.svg"></a>
+  <a href="#sponsors"><img src="https://opencollective.com/react-native-elements/sponsors/badge.svg"></a>
+  <a href="https://codecov.io/gh/react-native-elements/react-native-elements"><img src="https://codecov.io/gh/react-native-elements/react-native-elements/coverage.svg"></a>
+  <a href="https://github.com/prettier/prettier"><img src="https://img.shields.io/badge/styled_with-prettier-ff69b4.svg"></a>
+  <a href ="https://reactnativeelements.com/" ><img src="https://img.shields.io/badge/-React%20Native%20Elements-blue"/></a>
+  <a href="https://reactjs.org/"><img src="https://img.shields.io/badge/-React-blue"></a>
+  <a href="https://opensource.org/licenses/MIT"><img src="https://img.shields.io/badge/License-MIT-blue.svg"></a>
+  <a href="https://netlify.com"><img src="https://img.shields.io/badge/deployed%20on-netlify-blue" />  
+  
+</p>
+
+<br />
+<hr/>
+
+![React Native Elements UI Toolkit](https://raw.githubusercontent.com/tewarig/images/main/Screenshot%20from%202021-03-10%2019-44-18.png)
```

Comment:
Replace this with this -> https://github.com/react-native-elements/playground/blob/master/public/thumbnail.png

Reply:
ohk